src/main/java/com/banking/bankservice/repository/MySqlBankRepository.java

```diff
@@ -38,95 +32,58 @@ public Bank mapRow(ResultSet rs, int rowNum) throws SQLException {
 
     @Override
     public Bank find(int id) {
-        Connection con = dataBaseUtil.startConnection();
         try {
-            PreparedStatement pstmt = con.prepareStatement(String.format("select * from BANKS where id = %d", id));
-            ResultSet rs = pstmt.executeQuery();
-            if (rs.next()) {
-                String name = rs.getString("name");
-                String address = rs.getString("address");
-                return new Bank(id, name, address);
-            }
-            throw new EntityNotFoundException(String.format("Bank with ID = %d does not exist", id));
+            return jdbcTemplate.queryForObject(String.format(
+                    "select * from BANKS where id = %d", id), new RowMapper<Bank>() {
+                @Override
+                public Bank mapRow(ResultSet rs, int i) throws SQLException {
```

Comment:
This method here (mapRow) is duplicated in this class. Please, create a class for it.

Reply:
Done at last commit as private method within same class